onnxruntime/python/tools/transformers/models/llama/llama_parity.py

```diff
@@ -236,6 +240,14 @@ def get_args(argv: list[str]):
         choices=["int4", "int8", "fp16", "fp32"],
         help="Precision of model",
     )
+    
+    parser.add_argument(
+        "--dynamo",
+        action="store_true",
```

Comment:
Is the parity checker working with the Dynamo exported model? If not, then let's not make changes to this file.

Reply:
Sure, only thing is i did not modify this file at all.